src/http_tracker_connection.cpp

```diff
@@ -504,7 +504,7 @@ namespace libtorrent
 			else
 #endif
 			{
-				resp.peers4.reserve(len / 6);
+				resp.peers4.reserve(std::size_t(len / 6));
```

Comment:
thinking a bit about why this is a warning, presumably it's because ``len`` could be negative, which can't be represented by the unsigned variable. Maybe it would make sense to have a function perform these kinds of conversions, that also assert that the incoming value is not negative.